src/SFA.DAS.Funding.SystemAcceptanceTests/Helpers/FundingPeriodAssertions.cs

```diff
@@ -19,7 +19,7 @@ public class FundingPeriodAssertions : GenericCollectionAssertions<FundingPeriod
         {
             Execute.Assertion
                 .Given(() => subjArray[i])
-                .ForCondition(v => v.AgreedPrice*0.80m == targetValue)
+                .ForCondition(v => v.AgreedPrice == targetValue)
```

Comment:
logic change not devops code, is this intended to be in this PR?

Reply:
@tom-gough this is due to the problem with the commit history re-writing. This is supposed to be in main already